src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -277,6 +287,15 @@ def register_source_uri_arguments(scope):
 register_cli_argument('storage blob copy', 'blob_name', blob_name_type, options_list=('--destination-blob', '-b'), help='Name of the destination blob. If the exists, it will be overwritten.')
 register_cli_argument('storage blob copy', 'source_lease_id', arg_group='Copy Source')
 
+# BLOB INCREMENTAL COPY PARAMETERS
+register_blob_source_uri_arguments('storage blob incremental-copy start')
```

Comment:
What is the difference between incremental-copy and regular copy?  Do we now have incremental-copy, copy, and batch-copy?